charts/back8sup/templates/cronjob.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Capabilities.APIVersions.Has "batch/v1" }}
+{{- if .Capabilities.APIVersions.Has "batch/v1/CronJob" }}
```

Comment:
But wasn't this exactly the thing that we realized doesn't work with Argo CD?

Reply:
well we thought so, but i had this issue for a customer, and just tested it, it works like this... because `batch/v1` exists there but but for the `CronJob` kind.

Reply:
I remember looking this up in the Argo CD source code and coming to the conclusion that it doesn't support `<type>/<version>/<kind>`. For sure something here is inconsistent and needs more debugging given that this PR is basically a revert of #408.

Reply:
that's right, and it's totally a revert, which fixes our issue at the customer side

Reply:
oh and we use latest patch version of the application itself!

Reply:
yeah, but will the revert not break whatever was the reason for #408?

Reply:
well, i tested it with @isantospardo on the customer side, it works like this and we're running argocd v2.1.2 there!

Reply:
Yep,  I can confirm the fix on the customer side. Does anyone know any other customer running this? so we can test it there, from my side, I don't know any customer using it.

Reply:
so, approve/not approve?

Reply:
here you go:

```
~/s/helm/helm-charts/charts/back8sup | fix/back8sup-apiversion | $ h template --api-versions=batch/v1/CronJob . | grep '# Source: back8sup/templates/cronjob.yaml' -A2
# Source: back8sup/templates/cronjob.yaml
apiVersion: batch/v1
kind: CronJob
~/s/helm/helm-charts/charts/back8sup | fix/back8sup-apiversion | $ h template --api-versions=batch/v1 . | grep '# Source: back8sup/templates/cronjob.yaml' -A2       
# Source: back8sup/templates/cronjob.yaml
apiVersion: batch/v1beta1
kind: CronJob
~/s/helm/helm-charts/charts/back8sup | fix/back8sup-apiversion | $ h template --api-versions=batch/v1beta1/CronJob . | grep '# Source: back8sup/templates/cronjob.yaml' -A2
# Source: back8sup/templates/cronjob.yaml
apiVersion: batch/v1beta1
kind: CronJob
~/s/helm/helm-charts/charts/back8sup | fix/back8sup-apiversion | $ h template --api-versions=batch/v1beta1 . | grep '# Source: back8sup/templates/cronjob.yaml' -A2       
# Source: back8sup/templates/cronjob.yaml
apiVersion: batch/v1beta1
kind: CronJob
```

Reply:
totallly, it means that just `batch/v1` could not be having the `CronJob` in that `apiVersion`.
does it making sense?

Reply:
We might have seen how the template command is executed by the repo-server wrong. Le'ts see! LGTM

Reply:
cheerz, i'll roll that out to the customer!